src/utils/make-command.js

```diff
@@ -1,7 +1,34 @@
 const { EOL } = require( 'os' );
+const chalk = require( 'chalk' );
```

Comment:
Let's add a blank line between core node modules, external modules and internal modules for consistency.

```js
const { EOL } = require( 'os' );

const chalk = require( 'chalk' );
const logSymbols = require( 'log-symbols' );

const makeDocker = require( './make-docker' );
```